File: VB-Examples/10_Conversion/WordToMarkDown.vb
Imports Spire.Doc

Namespace WordToMarkDown
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Define the input file path relative to the current directory.
			Dim input As String = "..\..\..\..\..\..\Data\ToMD.docx"

			' Create a new document
			Dim doc As New Document()

			' Load .docx file
			doc.LoadFromFile(input)

			' Save to .md file
			doc.SaveToFile("WordToMarkDown_output.md", FileFormat.Markdown)

			' Dispose of the Document object
			doc.Close()

		
			WordDocViewer("WordToMarkDown_output.md")

		End Sub
		Private Sub WordDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace

File: CS-Examples/10_Conversion/WordToMarkDown.cs
using System;
using System.Windows.Forms;
using Spire.Doc;

namespace WordToMarkDown
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Define the input file path relative to the current directory.
            string input = @"..\..\..\..\..\..\Data\ToMD.docx";

            // Create a new document
            Document doc = new Document();

            // Load .docx file
            doc.LoadFromFile(input);

            // Save to .md file
            doc.SaveToFile("WordToMarkDown_output.md", FileFormat.Markdown);

            // Dispose of the Document object
            doc.Close();

     
            WordDocViewer("WordToMarkDown_output.md");

        }
        private void WordDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}
